_layouts/post.liquid

```diff
@@ -17,7 +17,7 @@
               by <a href="https://github.com/{{ author }}"><b>{{ contributors[author].name }}</b></a><em> - </em>
             {% elsif authors != null %}
               by 
-                {% assign members = authors | split: ", " %}
+                {% assign members = authors | split: "," %}
                   {% for member in members %}
```

Comment:
```suggestion
                  {% for member in authors %}
```

Now that `authors` is always an array (https://github.com/eslint/website/pull/895), I think we can just use it directly.